internal/app/render.go

```diff
@@ -64,10 +68,16 @@ func (a *App) renderDir(inputDirpath string, outputDirpath string, excluded []st
 
 		var output io.Writer = os.Stdout
 		var closer func()
-		if len(outputDirpath) > 0 {
-			if slices.Contains(excluded, filepath.Join(inputDirpath, relPath)) {
-				return nil
+		if slices.Contains(excludePaths, filepath.Join(inputDirpath, relPath)) {
+			return nil
+		} else {
+			for _, fg := range excludeFileGlobs {
+				if glob.MustCompile(fg).Match(relPath) || glob.MustCompile(fg).Match(filepath.Base(relPath)) {
```

Comment:
don't use MustCompile since it panics on error (see [source](https://github.com/gobwas/glob/blob/v0.2.3/glob.go#L54))
instead use Compile and ignore the err (like `foo, _ := glob.MustCompile`)

Reply:
ah missed that one, thx